### DISCUSSION TITLE
Mobile UI question

### DISCUSSION THREAD

**Discussion by niawag** [jcorporation/myMPD#693] [Q&A]
Hi and thanks for the many improvements that came with v9, you're doing a great work here !

Going from v8 to v9 I noticed a UI change, much appreciated on desktop but a bit less on mobile. As you can see here: 
![mympd v8 v9 mobile](https://user-images.githubusercontent.com/5684014/154838577-35521cc4-1fe7-4c7f-93aa-60537460256b.jpg)

Is there a setting I missed so that the UI on mobile looks like it does on desktop (tags as a single line, not multiple).
If there is stuff to test I'd be glad to help

Keep up the great work!

**Comment by jcorporation** [comment]
All tables fall back to tags in lines on small screens. There is no setting to change this behavior, but it is no problem to add a switch.

**Comment by niawag** [comment]
Well, thanks a lot, I'll try it as soon as it is out (not intended to put pressure) and report here!

**Comment by jcorporation** [comment]
There is now an option in the Settings -> Appearance -> Lists. Please check devel branch.

**Comment by niawag** [comment]
Perfect! Just tested it and it's working great, thanks a lot!

**Comment by dgcampea** [comment]
~~This option seems broken with Firefox on android, I get the V9 screen shown in the image above and the `Display tags in rows for small displays` option is enabled (mympd devel `sha256:1634207200ac28647b42c11d604066971d0e444e657f20228c1f1fe2fc398bc0`)~~

Edit: nvm, I misread the option as enable meaning V8 behavior.
